lib/activerecord_adbc_adapter/result.rb

```diff
@@ -37,6 +37,26 @@ def attach_model(model)
             when Arrow::StringDataType
               casted_type = Arrow::TimestampDataType.new(:nano)
             end
+          when :time
+            case field.data_type
+            when Arrow::StringDataType
+              # Parse time strings from SQLite's format (2000-01-01 HH:MM:SS.SSSSSS) and convert to Time64Array
+              # Format is defined in Rails' SQLite3 adapter:
+              # https://github.com/rails/rails/blob/90a1eaa1b30ba1f2d524e197460e549c03cf5698/activerecord/lib/active_record/connection_adapters/sqlite3/quoting.rb#L74-L77
+              time_values = column.data.chunks.flat_map do |chunk|
+                chunk.collect do |time_str|
+                  if time_str.nil?
+                    nil
+                  else
+                    dt = time_str.to_time
```

Comment:
Does `dt` stand for "date time"?
`time` may be better because we use `to_time` here.

BTW, should we refer `default_timezone` here? It seems that `String#to_time` uses local time by default.